app/views/directives/add-mobile-client.html

```diff
@@ -0,0 +1,36 @@
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title pull-left">Select Mobile Client to add to this service</h3>
+    <button class="panel-title btn pull-right" type="button" ng-click="ctrl.onClose()">
+      <span aria-hidden="true" class="pficon pficon-close"></span>
+    </button>
+    <div class="clearfix"></div>
+  </div>
+  <div class="panel-body">
+    <div class="list-pf">
+      <div ng-repeat="client in ctrl.clientsWhereExcluded">
```

Comment:
Can this `ng-repeat` just go on the `<div class="list-pf-item">` and then remove this otherwise unused div?

Reply:
Yeah, it can be removed. I'll update once I verify no difference :+1: